compress/test/cloud/test_run_gzip.py

```diff
@@ -0,0 +1,199 @@
+
+This sample demonstrates running a pipeline to compress a file that is in
+Google Cloud Storage.
+
+This sample demonstrates running the pipeline in an "ephemeral" manner;
+no call to pipelines.create() is neccessary. No pipeline is persisted
+in the pipelines list.
+
+Usage:
+  * python test_run_gzip.py \
+      --project <project-id> \
+      --disk-size <size-in-gb> \
+      --input <gcs-input-path> \
+      --output <gcs-output-path> \
+      --logging <gcs-logging-path> \
+      --poll-interval <interval-in-seconds>
+
+Where the poll-interval is optional (default is no polling).
+"""
+
+import argparse
+import pprint
+import time
+
+from oauth2client.client import GoogleCredentials
+from apiclient.discovery import build
+
+# Parse input args
+parser = argparse.ArgumentParser()
+parser.add_argument("--project", required=True,
+                    help="Cloud project id to run the pipeline in")
+parser.add_argument("--disk-size", required=True, type=int,
+                    help="Size (in GB) of disk for both input and output")
+parser.add_argument("--input", required=True,
+                    help="Cloud Storage path to input file")
+parser.add_argument("--output", required=True,
+                    help="Cloud Storage path to output file (with the .gz extension)")
+parser.add_argument("--logging", required=True,
+                    help="Cloud Storage path to send logging output")
+parser.add_argument("--poll-interval", default=0, type=int,
+                    help="Frequency (in seconds) to poll for completion (default: no polling)")
+args = parser.parse_args()
+
+# Create the genomics service
+credentials = GoogleCredentials.get_application_default()
+service = build('genomics', 'v1alpha2', credentials=credentials)
+
+# Run the pipeline
+operation = service.pipelines().run(body={
+  # The ephemeralPipeline provides the template for the pipeline
+  # The pipelineArgs provide the inputs specific to this run
+
+  # There are some nuances in the API that are still being ironed out
+  # to make this more compact.
+
+  'ephemeralPipeline' : {
+    'projectId': args.project,
+    'name': 'compress',
+    'description': 'Run "gzip" on a file',
+
+    # Define the resources needed for this pipeline.
+    'resources' : {
+      # Specify default VM parameters for the pipeline
+      'minimumCpuCores': 1,  # TODO: remove this when the API has a default
+      'minimumRamGb': 3.75, # TODO: remove this when the API has a default
+
+      # Create a data disk that is attached to the VM and destroyed when the
+      # pipeline terminates.
+      'disks': [ {
+        'name': 'data',
+        'autoDelete': True,
+
+        # Within the docker container, specify a mount point for the disk.
+        # The pipeline input argument below will specify that inputs should be
+        # written to this disk.
+        'mountPoint': '/mnt/data',
+
+        # Specify a default size and type
```

Comment:
Thanks- fixed them to be consistent with single-sentence comments omitting periods and mutli-sentence comments using periods.